app/javascript/react/components/Map/ControlPanel/ControlPanel.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react";
+import * as S from "./ControlPanel.style";
+import { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+import { setMapConfigId, setMapTypeId } from "../../../store/mapSlice";
+import { useAppDispatch } from "../../../store/hooks";
+import { RootState } from "../../../store";
+import { MapTypeId, ViewMode } from "../../../types/map";
+import { useTranslation } from "react-i18next";
```

Comment:
sort them please and add empty line between external and internal

Reply:
Do we have any configuration to do this on save? It would be useful 🙂